gaia-fast-list.js

```diff
@@ -386,7 +405,7 @@ Internal.prototype = {
    * @param  {HTMLElement} el
    * @param  {Number} i
    */
-  populateItem: function(el, i) {
+  populateItem: function(el, i, section) {
```

Comment:
the jsdoc is out of date (unless we remove the unused `section` param and the `return`) :)